cmd/spicedb/serve.go

```diff
@@ -315,16 +315,18 @@ func serveRun(cmd *cobra.Command, args []string) {
 		redispatch = remote.NewClusterDispatcher(v1.NewDispatchServiceClient(conn))
 	}
 
-	cachingRedispatch, err := caching.NewCachingDispatcher(redispatch, nil, "dispatch_client")
+	cachingRedispatch, err := caching.NewCachingDispatcher(nil, "local_caching")
```

Comment:
this isn't really `local` caching, right? metrics from this dispatcher will be requests that go through the dispatch api client?

Reply:
Changed back